core/src/main/java/zipkin2/reporter/Sender.java

```diff
@@ -51,6 +51,9 @@
   /**
    * Maximum bytes sendable per message including overhead. This can be calculated using {@link
    * #messageSizeInBytes(List)}
+   *
+   * Defaults to 500_000 bytes as conservative default given that Kafka default max.request.size is
```

Comment:
Don't think we need to tie to Kafka here either. How about this?

`Defaults to 500KB as a conservative default. You may get better or reduced performance by changing this value based on, e.g., machine size or network bandwidth in your infrastructure. Finding a perfect value will require trying out different values in production, but the default should work well enough in most cases.`